akka-docs/src/main/paradox/java/stream/stages-overview.md

```diff
@@ -76,20 +88,20 @@ separately from interval of the following ticks.
 
 ---------------------------------------------------------------
 
-### fromCompletionStage
+### fromFuture
 
-Send the single value of the `CompletionStage` when it completes and there is demand.
-If the `CompletionStage` fails the stream is failed with that exception.
+Send the single value of the `Future` when it completes and there is demand.
+If the future fails the stream is failed with that exception.
 
-**emits** when the `CompletionStage` completes
+**emits** the future completes
 
-**completes** after the `CompletionStage` has completed or when it fails
+**completes** after the future has completed
 
 ---------------------------------------------------------------
 
-### fromFuture
+### fromCompletionStage
 
-Send the single value of the Scala `Future` when it completes and there is demand.
+Send the single value of the @scala[`Future`] @java[`CompletionStage`] when it completes and there is demand.
```

Comment:
Not for this stage right? http://doc.akka.io/docs/akka/current/scala/stream/stages-overview.html#fromcompletionstage